cl_tf2/src/cl-tf2/stamped-data.lisp

```diff
@@ -70,17 +70,29 @@
  This call will:
    - define a class 'number-stamped' with slots 'num-value' and 'header'
    - define a constructor-function 'make-number-stamped'
+   - define a stamped converter 'number->number-stamped'
```

Comment:
Fixed in https://github.com/fairlight1337/roslisp_common/commit/c768f72044f0d95a555b8d1c200f2bab2ca7c56d.